pkg/routeagent_driver/main.go

```diff
@@ -99,6 +103,7 @@ func main() {
 		eventlogger.NewHandler(),
 		kubeproxy.NewSyncHandler(env.ClusterCidr, env.ServiceCidr),
 		ovn.NewHandler(&env, smClientset),
+		ovn.NewGatewayRouteHandler(&env, smClientset),
```

Comment:
Shouldn't we also add `ovn.NewNonGatewayRouteHandler`?

Reply:
Currently the new Handlers are not registered, but to be consistent, we should add it.

Reply:
added it , sorry it was missed while I created a selected a subset of changes.